src/app/component/CartItem/CartItem.component.js

```diff
@@ -137,17 +137,18 @@ export class CartItem extends PureComponent {
         );
     }
 
-    renderProductOptionValue = (optionValue) => {
+    renderProductOptionValue = (optionValue, i, array) => {
         const { label, value } = optionValue;
 
         return (
-            <div
+            <span
               block="CartItem"
               elem="ItemOptionValue"
               key={ label }
             >
                 { label || value }
-            </div>
+{ array[i + 1] ? <>, </> : null }
```

Comment:
What the hell is this monstrosity? 

```js
array[i + 1] ? <>, </> : null
```

1. Why to use fragments around the `, `?
2. This is unreadable, prefer variable like `isNextAvailable`
3. I am against conditional rendering, but in this case the `isNextAvailable && ', '` would work better.

Reply:
Yeah, jsx rule `react/jsx-one-expression-per-line` really wants to write this part like this.
I've made the isNextAvailabe variable, so should be better.